source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -0,0 +1,183 @@
+=================
+Change a Document
+=================
+
+.. default-domain:: mongodb
+
+In this page, you can learn how to change documents in a MongoDB collection
+using two distinct operation types: 
+
+- :ref:`Update <update-operation>`
+- :ref:`Replace <replace-operation>`
+
+Update operations change the value of the specified field in one or
+more documents using the 
+:java-docs:`updateOne() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#updateOne(org.bson.conversions.Bson,org.bson.conversions.Bson)>` 
+or :java-docs:`updateMany() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#updateMany(org.bson.conversions.Bson,org.bson.conversions.Bson)>` 
+method. A replace operation substitutes one document with a new
+specified document using the
+:java-docs:`replaceOne() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#replaceOne(org.bson.conversions.Bson,TDocument)>` 
+method.
+    
+Each method returns an ``AcknowledgedUpdateResult`` that tells us:
+
+- The number of documents that matched the query filter
+- The number of documents that got modified
+- The ``_id`` of the document that got upserted
+
+In the following examples, a paint store sells five different
+colors of paint. The ``paint_inventory`` collection represents their
+current inventory: 
+
+.. code-block:: json
+
+    { "_id": 1, "color": "red", "qty": 5 }
+    { "_id": 2, "color": "purple", "qty": 8 }
+    { "_id": 3, "color": "yellow", "qty": 0 }
+    { "_id": 4, "color": "green", "qty": 6 }
+    { "_id": 5, "color": "pink", "qty": 0 }
+
+.. _update-operation:
+
+Update
+------
+
+Update operations specify an update document to change in one or more
+documents that match your query filer using the 
+:java-docs:`updateOne() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#updateOne(org.bson.conversions.Bson,org.bson.conversions.Bson)>` 
+or :java-docs:`updateMany() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#updateMany(org.bson.conversions.Bson,org.bson.conversions.Bson)>` 
+method. 
+
+Update Operation Parameters:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Query filter: specifies the document you want to update in your collection
+- Update document: an :doc:`Updates builder </fundamentals/builders/updates>` that specifies the field and value to change in the document matched
+
+The update document uses the following format:
+
+.. code-block:: java
+
+    Bson update = Updates.<update operator>("<field>", "<value>");
+
+See the MongoDB API documentation for a :java-core-api:`complete list of
+update operators and their usage <com/mongodb/client/model/Updates.html>`.
+
+The ``updateOne()`` and ``updateMany()`` method uses the following format: 
+
+.. code-block:: java
+
+    collection.updateOne(<query filter>, <update document>);
+
+    collection.updateMany(<query filter>, <update document>);
```

Comment:
I think including a return type the way you're stating may cause confusion since I mention in the intro that each method returns an AcknowledgementResult